ranvier.json

```diff
@@ -15,29 +19,35 @@
     "core-quests",
     "debug-commands"
   ],
-
+  "excludedBuilderBundles": [
```

Comment:
This isn't necessary. `bundles` already acts as a whitelist for data loaded into the game and any data loaded into the game should be loaded into the API